### PR TITLE
align Sender's max msg size

### LINKED ISSUE #153: Reduce default message size
In docker testing, the default message size of 5MiB is too much.. it basically makes elasticsearch fall over. I think we should reduce significantly lower, like 1000000 bytes (kafka's size) or lower than that.

cc @anuraaga

### PR DESCRIPTION
**Pull request by jeqo**
Fix #153

### CODE REVIEW

File: activemq-client/src/main/java/zipkin2/reporter/activemq/ActiveMQSender.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public static Builder newBuilder() {
     ActiveMQConnectionFactory connectionFactory;
     String queue = "zipkin";
     Encoding encoding = Encoding.JSON;
-    int messageMaxBytes = 100000; // arbitrary to match kafka, messages theoretically can be 2GiB
+    int messageMaxBytes = 500_000; // arbitrary to match kafka, messages theoretically can be 2GiB
```
Comment (anuraaga) [inline activemq-client/src/main/java/zipkin2/reporter/activemq/ActiveMQSender.java]: If I'm not mistaken, this doesn't match Kafka anymore which is 1MB. Or maybe there's something related to bi-directionality here.

Could you explain more why we want to use half of Kafka max size as our default? If the reason is just to be somewhat conservative as a default and let users optimize themselves, it makes sense, but in that case I guess we don't need to mention Kafka in these comments.

Comment (jeqo) [reply activemq-client/src/main/java/zipkin2/reporter/activemq/ActiveMQSender.java]: @anuraaga yes, it's a conservative default as you mention. I will improve docs.

File: activemq-client/src/main/java/zipkin2/reporter/activemq/ActiveMQSender.java [hunk 2]
```diff
@@ -102,7 +102,7 @@ public Builder encoding(Encoding encoding) {
       return this;
     }
 
-    /** Maximum size of a message. Default 1000000. */
+    /** Maximum size of a message. Default 500KB. */
     public Builder messageMaxBytes(int messageMaxBytes) {
```
Comment (anuraaga) [inline activemq-client/src/main/java/zipkin2/reporter/activemq/ActiveMQSender.java]: Do you think we should update these builder methods to validate the size is less than the implementation limit (e.g., 1MB for kafka, 2GB for activemq)?

File: core/src/main/java/zipkin2/reporter/Sender.java [hunk 3]
```diff
@@ -51,6 +51,9 @@
   /**
    * Maximum bytes sendable per message including overhead. This can be calculated using {@link
    * #messageSizeInBytes(List)}
+   *
+   * Defaults to 500_000 bytes as conservative default given that Kafka default max.request.size is
```
Comment (anuraaga) [inline core/src/main/java/zipkin2/reporter/Sender.java]: Don't think we need to tie to Kafka here either. How about this?

`Defaults to 500KB as a conservative default. You may get better or reduced performance by changing this value based on, e.g., machine size or network bandwidth in your infrastructure. Finding a perfect value will require trying out different values in production, but the default should work well enough in most cases.`

File: activemq-client/src/main/java/zipkin2/reporter/activemq/ActiveMQSender.java (hunk 2 above)
Comment (jeqo) [reply activemq-client/src/main/java/zipkin2/reporter/activemq/ActiveMQSender.java]: not sure, as this limit can be customized (in Kafka at least)

**Comment by jeqo** [comment]
thanks @anuraaga for the review! I'll go ahead and merge this one.

**Pull request merged**